### PR TITLE
Latest amazonka from Hackage

### PR DESCRIPTION
**Pull request by EduardSergeev**

### CODE REVIEW

File: mismi-sqs/src/Mismi/SQS/Aws/Commands.hs [hunk 1]
```diff
@@ -90,18 +90,16 @@ toUserMessageAttributeValue v = do
...
-  u -> fail $ "Unsupported MessageAttribute: " <> unpack u
-
-toAttributes :: M.HashMap A.QueueAttributeName Text -> SQSAction [(MessageAttribute, T.Text)]
-toAttributes = traverse toPair . M.toList
-  where toPair (a, v) = (,) <$> toMessageAttribute (A.toText a) <*> pure v
+  A.All -> MessageAll
+  A.ApproximateFirstReceiveTimestamp -> ApproximateFirstReceiveTimestamp
+  A.ApproximateReceiveCount -> ApproximateReceiveCount
+  A.SenderId -> SenderId
+  A.SentTimestamp -> SenderId
```
Comment (charleso) [inline mismi-sqs/src/Mismi/SQS/Aws/Commands.hs]: Is this right? In any case can/should we remove the `Aws` module fairly soon?

Comment (EduardSergeev) [inline mismi-sqs/src/Mismi/SQS/Aws/Commands.hs]: No, it is a mistake. (Un)fortunately this code is never run and will probably going to be removed soon. Will fix it meanwhile.

**Comment by EduardSergeev** [comment]
@charleso @nhibberd @markhibberd Latest `amazonka` seems to pass all tests. Worth merging? This change includes its removal from submodules (I believe it is now supposed to be fetched from hackage) - do we want that? Or should we stick with git repo (due UTCTime problems @charleso discovered)?

**Comment by charleso** [comment]
@EduardSergeev As discussed IRL, it can't hurt to do an upgrade and production test of the usual suspects (eg. arm, juice and sanction). 

I vote for removing the submodule, we can always add/force it back in. But I'll let you know if the date bug I've found is going to affect us in the short term.

**Comment by EduardSergeev** [comment]
Tests *were* passing before rebase. Now `prop_list` is failing due to duplicated results. Which is apparently cased by the changes in `amazonka` - the changes are now causing recently added `list'` function to misbehave (if old `listObjects`-based mechanism is used instead the test is passing). @nhibberd could you maybe quickly spot the problem?

**Comment by markhibberd** [comment]
I am :100: for removing the submodule, we should do that as soon as is possible. But I lack a bit of context on the other issues:
 - Broken pagination / listing would be very bad, so obviously needs to be looked at first.
 - What is the UTCTime issue and what does it affect?

**Comment by charleso** [comment]
> What is the UTCTime issue and what does it affect?

The `ISO8601` format seems to be broken, at least for CloudWatch. It may have always been this way. I'm about to submit a patch.

**Comment by nhibberd** [comment]
I'll have a look at the listing stuff this morning

**Comment by nhibberd** [comment]
https://github.com/brendanhay/amazonka/issues/218

**Comment by brendanhay** [comment]
Bloops! :full_moon_with_face: 

Version [1.3.1](http://hackage.haskell.org/package/amazonka-1.3.1) is now available on Hackage with the fixes to the ListObjects pagination and the ISO8601 fixes from @charleso.

**Comment by nhibberd** [comment]
Awesome! Nice work @brendanhay

File: lib/amazonka [hunk 2]
```diff
@@ -1 +0,0 @@
-Subproject commit 55298d0ea60c0214cd001130f51731f93b49ed5f
```
Comment (charleso) [inline lib/amazonka]: :100: :100: :100: :100: :100: :100: :100:

**Pull request merged**